.github/workflows/superset-python-integrationtest.yml

```diff
@@ -150,7 +150,9 @@ jobs:
       SUPERSET_CONFIG: tests.integration_tests.superset_test_config
       REDIS_PORT: 16379
       SUPERSET__SQLALCHEMY_DATABASE_URI: |
-        sqlite:///${{ github.workspace }}/.temp/unittest.db
```

Comment:
Weird name for a database used for integration tests. Calling this `superset.db` is inline with how the metadata database is used elsewhere.